dask/bag/core.py

```diff
@@ -1956,7 +1956,7 @@ def map_partitions(func, *args, **kwargs):
             args2.append((itertools.repeat, a.key))
```

Comment:
This is likely broken as well. It looks like you copied this code over from `bag.map`, which does support these options.

Reply:
Thanks for the catch.  Tested and fixed.